superdesk/io/commands/remove_expired_content.py

```diff
@@ -36,25 +33,31 @@ def run(self, provider_name=None):
                 self.remove_expired(provider)
 
     def remove_expired(self, provider):
+        lock_name = 'ingest:gc'
+
+        if not lock(lock_name, expire=300):
```

Comment:
looking at the library code, the host doesn't matter much. it's only used when you would like to extend the lock, but there can be only 1 lock with given name